backend/tracim_backend/views/core_api/user_controller.py

```diff
@@ -238,6 +238,7 @@ def set_user_infos(self, context, request: TracimRequest, hapic_data=None):
             lang=hapic_data.body.lang,
             do_save=True
         )
+        uapi.execute_updated_user_actions(user)
```

Comment:
remove this

Reply:
see : https://github.com/tracim/tracim/pull/1578#discussion_r274287675

Reply:
https://github.com/tracim/tracim/issues/1589